.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: Erlang CI
+on: push
+jobs:
+  Test:
+    strategy:
+      matrix:
+        erlang: ['24']
```

```suggestion
        erlang: ['24', '26']
```
We are almost done with the migration to OTP 26, so let's start testing OTP 26 too.